Uno.Gallery/Uno.Gallery.UITest/Constants.cs

```diff
@@ -9,7 +9,7 @@ namespace Uno.Gallery.UITests
 {
     public class Constants
     {
-        public readonly static string WebAssemblyDefaultUri = "http://localhost:64555/";
+        public readonly static string WebAssemblyDefaultUri = "http://localhost:56745/";
```

Comment:
The port should be fixed using a `launchSettings.json` file [like we did on the sample app](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.Wasm/Properties/launchSettings.json).

Reply:
There's still a port define in the [Constants.cs](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Constants.cs) no?

Reply:
Yes... but why are you changing it?

Reply:
It didn't match the port in the launchSettings and was not launching the tests for WASM properly